File: src/#1078/solution.go
package solution

import "strings"

func findOcuurences(text string, first string, second string) []string {
	newStr := strings.Split(text, " ")
	res := []string{}

	if len(newStr) < 2 {
		return res
	}

	for i := 2; i < len(newStr); i++ {
		if newStr[i-2] == first && newStr[i-1] == second {
			res = append(res, newStr[i])
		}
	}
	return res
}

File: src/#1078/solution.js
var findOcurrences = function (text, first, second) {
  let newStr = text.split(' ')
  let res = []

  if (newStr.length < 2) {
    return res
  }

  for (let i = 2; i < newStr.length; i++) {
      if (newStr[i-2]===first&&newStr[i-1]===second){
        res.push(newStr[i])
      }
  }
  return res
}
